bin/civi-test-run

```diff
@@ -179,6 +189,7 @@ for TESTTYPE in $SUITES ; do
     phpunit-crm)  task_phpunit CRM_AllTests ;;
     phpunit-api)  task_phpunit api_v3_AllTests ;;
     phpunit-civi)  task_phpunit 'Civi\AllTests' ;;
+    phpunit-drupal) phpunit_drupal ;;
```

Comment:
@seamuslee001 Nice! I'd really want to get the `civicrm-drupal` suite running.

Minor nitpick: the other functions have a naming convnetion (`task_karma`, `task_upgrade`, `task_phpunit`). This one should be `task_phpunit_drupal`.

Reply:
changed in the latest commit @totten